packages/zora/src/Zora.test.ts

```diff
@@ -0,0 +1,34 @@
+import { apply } from '@rabbitholegg/questdk/filter'
+import { describe, expect, test } from 'vitest'
+import { failingTestCases, passingTestCases } from './test-setup'
+import { mint } from './Zora'
+
+describe('Given the zora plugin', () => {
+  describe('When handling the mint', () => {
+
+    test('should return a valid action filter', () => {
+      
+    })
```

Comment:
Still need this?

Reply:
I should at a filter test - yeah good callout